### DISCUSSION TITLE
Canva

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47313] [GitHub Education] [answered]
### Select Topic Area

Product Feedback

### Body

Is canva still part of Github's student developer pack?

**Comment by thuythanh95** [comment]
ok

**Comment by radar-blip** [comment, marked as answer]
See answer here: https://github.com/orgs/community/discussions/28136#discussioncomment-3319262

Also here: #46144

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Mar080901! We're happy you're here. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Cheers!
